recipe/meta.yaml

```diff
@@ -21,23 +21,25 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python >=3.6
-    - ipython >=4.0.0
+    - python
+    - ipython >=6.1.0
     - ipykernel >=4.5.1
-    - traitlets >=4.3.1,<6.0.0
+    - traitlets >=4.3.1
     - nbformat >=4.2.0
```

Comment:
nbformat was removed, see https://github.com/jupyter-widgets/ipywidgets/commit/04d42174b3580f1f1b44ced0c79f4c3e60ad4084 and https://github.com/jupyter-widgets/ipywidgets/blob/8.0.4/python/ipywidgets/setup.cfg#L36-L41

Reply:
done